### PR TITLE
Minor rough spots.

### PR DESCRIPTION
**Pull request by m-reza-rahman**

### CODE REVIEW

**Comment by rjeberhard** [comment]
@m-reza-rahman, have you signed the contributor agreement or is Microsoft a signatory? The agreement is here: https://github.com/oracle/weblogic-kubernetes-operator#contributing-to-the-oracle-weblogic-server-kubernetes-operator-repository

File: docs-source/content/samples/simple/azure-kubernetes-service/_index.md [hunk 1]
```diff
@@ -123,10 +123,11 @@ versions may work as well.
 
 ##### Clone WebLogic Server Kubernetes Operator repository
 
-Clone this repository to your machine. We will use several scripts in this repository to create a WebLogic domain. This sample was tested with v3.0.0.
+Clone the [Oracle WebLogic Server Kubernetes Operator repository](https://github.com/oracle/weblogic-kubernetes-operator) to your machine. We will use several scripts in this repository to create a WebLogic domain. This sample was tested with v3.0.0.
 
 ```bash
 $ git clone https://github.com/oracle/weblogic-kubernetes-operator.git
+#cd weblogic-kubernetes-operator
```
Comment (rjeberhard) [inline docs-source/content/samples/simple/azure-kubernetes-service/_index.md:130]: Is it intentional that the `cd` is commented?

**Comment by edburns** [comment]
> @m-reza-rahman, have you signed the contributor agreement or is Microsoft a signatory? The agreement is here: https://github.com/oracle/weblogic-kubernetes-operator#contributing-to-the-oracle-weblogic-server-kubernetes-operator-repository

Hello @rjeberhard , here's the plan.  @m-reza-rahman will keep iterating in this PR and when it is ready, I will clone the PR to my GitHub, close this one, and I'll re-issue a new PR from there.  I have already signed the OCR (Ed Bratt can confirm).

**Review by mriccell — APPROVED** [review]

**Comment by edburns** [comment]
Please close this PR. Cc: @m-reza-rahman  @rjeberhard  @mriccell   See #1940.

**Comment by rjeberhard** [comment]
This PR is superseded.

**Pull request closed without merging**